sources/packages/backend/apps/api/src/route-controllers/application-change-request/_tests_/e2e/application-change-request.aest.controller.pendingApplicationChangeRequest.e2e-spec.ts

```diff
@@ -0,0 +1,336 @@
+      appDataSource,
+      undefined,
+      {
+        initialValues: {
+          applicationStatus: ApplicationStatus.Completed,
+          applicationEditStatus: ApplicationEditStatus.Original,
+        },
+      },
+    );
+
+    // Create another application with ChangeDeclined status
+    const declinedApplication = await saveFakeApplication(
+      appDataSource,
+      undefined,
+      {
+        initialValues: {
+          applicationStatus: ApplicationStatus.Edited,
+          applicationEditStatus: ApplicationEditStatus.ChangeDeclined,
+        },
+      },
+    );
+
+    const token = await getAESTToken(AESTGroups.BusinessAdministrators);
+    const endpoint =
+      "/aest/application-change-request/pending?page=0&pageLimit=10&sortField=submittedDate&sortOrder=DESC";
+
+    // Act
+    const response = await request(app.getHttpServer())
+      .get(endpoint)
+      .auth(token, BEARER_AUTH_TYPE)
+      .expect(HttpStatus.OK);
+
+    // Assert
+    expect(Array.isArray(response.body.results)).toBe(true);
+
+    const foundNonPending = response.body.results.find(
+      (item: any) => item.applicationId === nonPendingApplication.id,
+    );
+    const foundDeclined = response.body.results.find(
+      (item: any) => item.applicationId === declinedApplication.id,
+    );
+
+    expect(foundNonPending).toBeUndefined();
+    expect(foundDeclined).toBeUndefined();
+  });
+
+  it("Should handle pagination parameters correctly", async () => {
+    // Arrange
+    const token = await getAESTToken(AESTGroups.BusinessAdministrators);
+    const endpoint5 =
+      "/aest/application-change-request/pending?page=0&pageLimit=5&sortField=submittedDate&sortOrder=DESC";
+    const endpoint10 =
+      "/aest/application-change-request/pending?page=0&pageLimit=10&sortField=submittedDate&sortOrder=DESC";
+
+    // Act - Test with different page sizes
+    const responsePageSize5 = await request(app.getHttpServer())
+      .get(endpoint5)
+      .auth(token, BEARER_AUTH_TYPE)
+      .expect(HttpStatus.OK);
+
+    const responsePageSize10 = await request(app.getHttpServer())
+      .get(endpoint10)
+      .auth(token, BEARER_AUTH_TYPE)
+      .expect(HttpStatus.OK);
+
+    // Assert
+    expect(responsePageSize5.body.results.length).toBeLessThanOrEqual(5);
+    expect(responsePageSize10.body.results.length).toBeLessThanOrEqual(10);
+
+    // Test pagination offset
+    if (responsePageSize10.body.count > 10) {
+      const endpointPage1 =
+        "/aest/application-change-request/pending?page=1&pageLimit=10&sortField=submittedDate&sortOrder=DESC";
+      const responsePage1 = await request(app.getHttpServer())
+        .get(endpointPage1)
+        .auth(token, BEARER_AUTH_TYPE)
+        .expect(HttpStatus.OK);
+
+      expect(responsePage1.body.results).toBeDefined();
+      expect(Array.isArray(responsePage1.body.results)).toBe(true);
+    }
+  });
+
+  it("Should return empty results when no pending change requests exist", async () => {
```

Comment:
The following test here validates empty result. https://github.com/bcgov/SIMS/pull/4733/files#r2107888347 Hence this test can be removed.